packages/gestalt/src/Stack.js

```diff
@@ -39,37 +42,23 @@ type Props = {|
   wrap?: boolean,
 |};
 
-export default function Stack({
-  alignItems = 'start',
-  children,
-  gap = 0,
-  height,
-  justifyContent = 'center',
-  width,
-  ...rest
-}: Props): Node {
+// Defaults for align-items and justify-content set in base styles
+export default function Stack({ children, gap = 0, ...rest }: Props): Node {
+  const baseClasses = classnames(
+    styles.stack,
+    styles[`verticalGap${gap}`],
+    styles[`verticalGap${gap} > *`]
```

Comment:
@dangerismycat do we need this here? I think we should be able to remove it since `verticalGap > *` is not a valid classname.

Reply:
agreed, removed